dynamodb/query.py

```diff
@@ -38,6 +43,46 @@ def get_patient(self, nhs_number, parameters=None):
         else:
             return None
 
+    def get_immunisation_by_id(self, imms_id):
+        response = self.table.get_item(
+            Key={'PK': f"Immunisation#{imms_id}"},
```

Comment:
This and all other usages of `Immunisation#` could be a variable rather than repeated. Removes typo errors and explains what it is by variable name.